packages/playwright/src/generators/configuration/files/playwright.config.ts.template

```diff
@@ -0,0 +1,77 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './tests',
```

Comment:
this might need to be dynamic, like the cypress project one to either be `e2e` if adding to an existing project or `src` if being added to an 'empty' project that is only for e2e testing or we could always use 'e2e'. 

Personally not a fan of 'tests' as it's unclear if it's unit tests/playwright tests without looking at the tests.

Reply:
add an option in the schema called "directory" to allow developers to specify the test directory